nixos/modules/programs/ccache.nix

```diff
@@ -28,7 +28,9 @@ in {
 
       # "nix-ccache --show-stats" and "nix-ccache --clear"
       security.wrappers.nix-ccache = {
+        owner = "nobody";
```

Comment:
We should have a dedicated user - not a shared "nobody" user.

Reply:
I'm trying to preserve the current permissions/ownership in this PR. I agree it's better to avoid nobody, but I don't want risking breaking program or modules.

Reply:
cc @kira-bruneau @r-burns as maintainers of ccache

Reply:
Yep, I think it makes sense to add a dedicated user. @r-burns and I didn't originally write this (we made ourselves maintainers after noticing it was abandoned), but it looks like as long as the group stays as `nixbld`, there shouldn't be any problems with permissions.

I'll try it out the change to be sure.

Reply:
It looks like volth originally wrote the module, but I just relalized they don't have a github account anymore :disappointed:

Reply:
They still have an account on the nixos.org forum, if you want to try.